lib/public_key/src/pubkey_cert.erl

```diff
@@ -2195,3 +2196,20 @@ otp_cert(#'OTPCertificate'{} = Cert) ->
     Cert;
 otp_cert(#cert{otp = OtpCert}) ->
     OtpCert.
+
+key_params(#'OTPCertificate'{tbsCertificate = TBSCert}, Params) when Params == asn1_NOVALUE;
+                                                                     Params == 'NULL' ->
+    #'OTPTBSCertificate'{signature =
+                             #'SignatureAlgorithm'{algorithm = Algo,
+                                                   parameters = KeyParams}
+                        } = TBSCert,
+    %% Sometimes parameters may be missing in issuer's "SubjectPublicKeyInfo" but included in
+    %% the certs "SignatureAlgorithm" for RSA PSS signatures.
+    case Algo of
+        ?'id-RSASSA-PSS' ->
+            KeyParams;
+        _ ->
+            Params
+    end;
+key_params(_, KeyParams) ->
```

Comment:
Use same `Params` variable name for 2nd argument as in previous function clause?
`KeyParams` refers to different things between function clauses. maybe if avoided, reading would easier .